kolibri/plugins/media_player/assets/src/mixins/videojsMenuVueMixin.js

```diff
@@ -27,6 +27,21 @@ export default function videojsMenuVueMixin(vueComponent) {
       return this.getVueComponent().contentEl();
     }
 
+    /**
+     * `contentEl` is used when `addItem` is called, so this allows the addition of the text track
+     * options (the languages) in the right spot
+     *
+     * @override
+     * @return {*|Element}
+     */
+    get contentEl_() {
+      return this.contentEl();
+    }
+
+    set contentEl_(value) {
+      // No op - this gets called by VideoJS but we don't want it to mess with our DOM in this way.
+    }
+
```

Comment:
Yeah looking at [this](https://github.com/videojs/video.js/blob/v7.20.2/src/js/menu/menu.js#L118), these changes make sense!